### PR TITLE
chore: fix wrong essential param to optional

### PR DESCRIPTION
**Pull request by yukkysaito**
## PR Type

- Bug Fix

## Description
Fixed a parameter that was written in the wrong line.

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Code follows [coding guidelines][coding-guidelines]
- [x] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [x] Commits are properly organized and messages are according to the guideline
- [x] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [x] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR**: Required to pass before the merge.
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: autoware_launch/launch/autoware.launch.xml [hunk 1]
```diff
@@ -4,10 +4,10 @@
   <arg name="map_path" description="point cloud and lanelet2 map directory path"/>
   <arg name="vehicle_model" description="vehicle model name"/>
   <arg name="sensor_model" description="sensor model name"/>
-  <arg name="use_pointcloud_container" default="true" description="launch pointcloud container"/>
   <arg name="pointcloud_container_name" default="pointcloud_container"/>
```
Comment (kenji-miyake) [inline autoware_launch/launch/autoware.launch.xml]: Should `pointcloud_container_name` be optional as well? :thinking:

Comment (yukkysaito) [reply autoware_launch/launch/autoware.launch.xml]: ops...
Yes, you are correct.

Comment (yukkysaito) [reply autoware_launch/launch/autoware.launch.xml]: https://github.com/tier4/autoware_launch/pull/197/commits/6165a30180b96e9f9b015f8a9a9b6fe67ad318e7

**Review by kenji-miyake — APPROVED** [review]
BTW, I edited the PR title for conventional commits.
Please re-edit it if you prefer another one. :bow:

**Pull request merged**